LdapForNet/Native/NativeMethodsLinux.cs

```diff
@@ -6,8 +6,8 @@ namespace LdapForNet.Native
 {
     internal static class NativeMethodsLinux
     {
-        private const string LIB_LDAP_PATH = "ldap-2.4.so.2";
-        private const string LIB_LBER_PATH = "lber-2.4.so.2";
+        private const string LIB_LDAP_PATH = "ldap.so.2";
```

Comment:
please add notes to documentation about OpenLDAP library finding on the Linux platform. So a user should create symlink "ldap.so.2" and "lber.so.2" If some Linux distributive does not have such symlink